package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "awesome-books",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
[REQUIRED] I am also finding it hard to give a proper review because you have no scripts to run your code in a browser and there are no instructions in your README on how to go about it.

Reply:
Thanks @fobadara for this review. Please I have a little concern. Following the instructions and requirements that was given to us in this project, I didn't come across any that highlighted the need to include a script command in package.json file to run this code in the browser. This is the first time I am receiving such review. How am I supposed to do this without using a 3rd party package? Thanks

Reply:
You are welcome. The major issue is running your code in a browser. Since you are using webpack you should have come across webpack dev server. How are you running your code in the browser?

Reply:
You should have come across something like this:
` "scripts": {
    "build": "webpack",
    "dev": "webpack serve"
  },`

Reply:
Thanks @fobadara for your swift reply. However, I haven't come across using webpack dev server. What I usually do is use a live server on vs code to deploy my index.html file. I guess this would require me to install an extra package(webpack)?

Reply:
@AyoMoses1, This is a pair programming project right, 🤔 But I don't see any commits from your partner !!

Reply:
Please find below a screenshot of the email
![Screenshot (291)](https://user-images.githubusercontent.com/53434382/213252606-8a2eb3d3-6588-4c53-b17f-e31a9b2c6ddd.png)